cli/compose/loader/volume.go

```diff
@@ -23,12 +26,13 @@ func parseVolume(spec string) (types.ServiceVolumeConfig, error) {
 	}
 
 	buffer := []rune{}
-	for _, char := range spec {
+	for _, char := range spec + string(endOfSpec) {
```

Comment:
Interesting. I wonder if it would make sense to iterate over bytes instead of runes, so that we could just check if the iteration count is equal to the length of the sequence minus one. I don't think Windows drive letters can be non-ASCII, but I could be wrong about this.